pages/getData.tsx

```diff
@@ -22,6 +22,10 @@ const dataKeyList = [
   ...LSP6DataKeys.map((key) => ({ name: key.name, key: key.key, icon: '🔑' })),
   ...LSP9DataKeys.map((key) => ({ name: key.name, key: key.key, icon: '🔒' })),
 ];
+const ADDDRESS_PERMISSIONS_PERMISSION_PREFIX = dataKeyList[17].key.substring(
```

Comment:
I do not suggest to use index 17 hardcoded - if we add an element in the list above, it will break this.

You can find indexOf instead:

```
LSP6DataKeys.indexOf('AddressPermissions:Permissions:<address>')
```